assets/grid-collapsed.css

```diff
@@ -0,0 +1,51 @@
+/* 
```

Comment:
This file will be.. a lot. I figure it might be worthwhile to at least define some of the vars being used, but I'm happy to remove them if they're not helpful.